src/marvin/settings.py

```diff
@@ -44,8 +44,9 @@ def __setattr__(self, name: str, value: Any) -> None:
 
 class ChatCompletionSettings(MarvinSettings):
     model_config = SettingsConfigDict(env_prefix="marvin_chat_completion_")
-    model: str = Field(
-        description="The default chat model to use.", default="gpt-3.5-turbo"
+    model: str = Field(description="The default chat model to use.", default="gpt-4")
```

Comment:
```suggestion
    model: str = Field(description="The default chat model to use.", default="gpt-4-1106-preview")
```

I think if you have GPT-4 you have the turbo preview, so this is better, cheaper and much larger context

Reply:
updated in [a657902](https://github.com/PrefectHQ/marvin/pull/746/commits/a657902cacffe49be67e3f20e68f49177d15db84)